types/wordpress__notices/index.d.ts

```diff
@@ -62,7 +96,7 @@ export interface Options {
      * The type of notice.
      * @defaultValue `'default'`
      */
-    type: 'default' | 'snackbar';
+    type: WPNoticeType;
```

Comment:
```suggestion
    type: NoticeType;
```
(same reason as in other place)